### PR TITLE
test: Setup CI

### PR DESCRIPTION
**Pull request by jcs090218**
Also fixes the URL.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	@echo "Installing..."
+	$(EASK) install
+
+compile:
+	@echo "Compiling..."
+	$(EASK) compile
+
+test:
+	@echo "Testing..."
+	$(EASK) test ert ./test/*.el
```
Comment (Fuco1) [inline Makefile]: We are using buttercup not ert

**Review by Fuco1 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jcs090218) [reply Makefile]: Updated!

**Comment by jcs090218** [comment]
I've completely removed Makefile and changed the test from ert to buttercup.

**Comment by Fuco1** [comment]
Great, thanks!

**Pull request merged**